webapp/sources/rudder/rudder-rest/src/main/scala/com/normation/rudder/rest/lift/ComplianceApi.scala

```diff
@@ -412,12 +416,13 @@ class ComplianceApi(
  * compliance for all rules/nodes/directives.
  */
 class ComplianceAPIService(
-    rulesRepo:                   RoRuleRepository,
-    nodeInfoService:             NodeInfoService,
-    nodeGroupRepo:               RoNodeGroupRepository,
-    reportingService:            ReportingService,
-    directiveRepo:               RoDirectiveRepository,
-    val getGlobalComplianceMode: () => Box[GlobalComplianceMode]
```

Comment:
are you sure it's not used in a plugin or something? It's strange enough to have a val here to be cautious

Reply:
ok, I didn't find anything :shrug:

Reply:
I also didn't find anything :smile_cat: